### PR TITLE
Seedlet: Add underline to comment author/edit links on hover

### LINKED ISSUE #3330: Seedlet: Comment author/edit links need an underline on hover
In Seedlet, the comment action links currently do not have a visible indication that they've been hovered: 

Seedlet-Comments-Hover

These should use the same `border-bottom` border treatment we use for other links throughout the theme: 

Screen Shot 2021-02-22 at 3 03 43 PM

### PR DESCRIPTION
**Pull request by danieldudzic**
Closes #3330

### CODE REVIEW

File: seedlet/assets/sass/style.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ Description: Seedlet is a free WordPress theme. A two-column layout and classica
 Requires at least: 5.5
 Tested up to: 5.5
 Requires PHP: 5.6.2
-Version: 1.1.11-wpcom
+Version: 1.1.10-wpcom
```
Comment (scruffian) [inline seedlet/assets/sass/style.scss]: Shouldn't this go up?

File: seedlet/assets/sass/components/comments/_comment-form.scss [hunk 2]
```diff
@@ -23,6 +23,11 @@
 	.url {
 		border-bottom: none;
```
Comment (scruffian) [inline seedlet/assets/sass/components/comments/_comment-form.scss:24]: I wonder if we should just remove this line, so the behaviour is the same as on the rest of the theme, i.e. the link is underlined until it's hovered.

**Review by scruffian — CHANGES_REQUESTED** [review]
I think we should make the behaviour consistent with the rest of the theme, and hide the underline on hover, but we should probably get a second opinion from @kjellr

**Review by kjellr — COMMENTED** [review]
> I think we should make the behaviour consistent with the rest of the theme, and hide the underline on hover, but we should probably get a second opinion from @kjellr

I prefer it this way (underline only appears on hover) for utility links that appear close together like these ones do. We already do this for post meta. 👍 

The only issue I see in this PR is that the author link should be green on hover, just like the other links: 

hover

Once that's fixed, this should be good to go.

**Comment by kjellr** [comment]
I just pushed a quick update sync up with `trunk` and also cover that text color:

hover

Should be all set, if someone can give it one last review.

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**